numerics.tex

```diff
@@ -70,8 +70,6 @@ \subsection{The mpmath library}
 >>> import mpmath
 >>> mpmath.mp.dps = 30    # 30 digits of precision
 >>> mpmath.mpf("0.1") + mpmath.exp(-50)
-mpf('0.100000000000000000000192874984794')
->>> print(_)   # pretty-printed
```

Comment:
I don't like this change. This loses some information that was portrayed here. And this *is* the output:

```py
>>> import mpmath
>>> mpmath.mp.dps = 30    # 30 digits of precision
>>> mpmath.mpf("0.1") + mpmath.exp(-50)
mpf('0.100000000000000000000192874984794')
>>> print(mpmath.mpf("0.1") + mpmath.exp(-50))
0.100000000000000000000192874985
```